go/oasis-test-runner/scenario/e2e/gas_fees_staking.go

```diff
@@ -106,8 +148,13 @@ func (sc *gasFeesImpl) Run(childEnv *env.Env) error {
 		return err
 	}
 
+	// Include fees from genesis.
+	totalFees, err := sc.getInitialCommonPoolBalance(ctx)
```

Comment:
does this require that the entire common pool come from fees or something?

Reply:
no doesn't matter, in any case common pool will get distributed to nodes. Updated the comment to not explicitly mention fees and also updated the staking-genesis doc used to have both non-empty initial `last_block_fees` and `common_pool`